<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxtFilesForm</class>
 <widget class="QWidget" name="TxtFilesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(222,232,235);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBoyStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBoyEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxGirlStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxGirlEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLastStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLastEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCountryStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCountryEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxReligionStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxReligionEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSportStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSportEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxProfessionStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxProfessionEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Boys Names (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Girls Names (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Girls Names (End)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Last Names (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Last Names (End)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Countries (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Countries (End)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Boys Names (End)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Religions (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Religions (End)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Sports (End)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Sports (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Professions (Start)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Professions (End)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
